src/client/public/scripts/components/nav-element.ts

```diff
@@ -6,20 +6,29 @@ import {BaseElement, property} from './base-element.js';
 
 export class NavElement extends BaseElement {
   @property() data: api.UserResponse|null = null;
+  @property() documentUrl = document.location.pathname;
```

Comment:
Add a comment here explaining this is just to take advantage of property binding.

Reply:
Done.